src/cdk/text-field/autosize.ts

```diff
@@ -58,6 +62,10 @@ export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
     this._setMaxHeight();
   }
 
+  /** Emit event when textarea height changes */
+  @Output('cdkAutosizeSizeChanged')
+  sizeChanged = new EventEmitter<number>(false);
```

Comment:
Even though the type is inferred, this one still needs `: EventEmitter<number>` in order for it to show up correctly in the docs.